shuttle/driver_logs/__init__.py

```diff
@@ -36,3 +38,41 @@ def selected_shuttle_logs(self):
         completed_requests = selected_logs[2]
         deleted_requests = selected_logs[3]
         return render_template('driver_logs/load_logs.html', **locals())
+
+
+    @route('/download-logs/<date>/<name_sort>', methods=['GET', 'POST'])
+    def export_logs(self, date, name_sort):
+        selected_logs = self.ssc.grab_selected_logs(date, name_sort)
+        current_date = datetime.now().strftime('%m/%d/%Y')
+        date = datetime.strptime(date, '%b-%d-%Y').strftime('%m/%d/%y')
+        csv_name = 'driver_logs_' + date
+        log_type = ['Shuttle Logs', 'Break Logs', 'Completed Logs', 'Deleted Logs']
+
+        def generate():
+            # sets up a buffer and start file
+            data = StringIO()
+            writer = csv.writer(data)
+            new_line = []
+            writer.writerow([csv_name, 'Exported on: ' + current_date])
+            writer.writerow(new_line)
+
+            for i in range(len(selected_logs)):
+                for j in range(len(selected_logs[i])):
+                    # grabs a log, adds its data if its not empty
+                    log_dict = selected_logs[i][j]
+                    if log_dict:
+                        dict_writer = csv.DictWriter(data, fieldnames=log_dict.keys())
+                        if j == 0:
+                            writer.writerow([log_type[i]])
+                            dict_writer.writeheader()
+                        dict_writer.writerow(log_dict)
+                        yield data.getvalue()
+                        data.seek(0)
+                        data.truncate(0)
+                writer.writerow(new_line)
+            data.close()
```

Comment:
Can you change it so that you can clearly see where a new log begins. 

Maybe change this:
![Screen Shot 2020-08-28 at 1 21 26 PM](https://user-images.githubusercontent.com/26125143/91602808-91fa7100-e931-11ea-93d9-2989a78544e6.png)

To this:
![Screen Shot 2020-08-28 at 1 22 14 PM](https://user-images.githubusercontent.com/26125143/91602837-99217f00-e931-11ea-8068-8606b255f65a.png)